index.html

```diff
@@ -303,25 +303,62 @@ <h2><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> States a
-			<li><rref>listbox</rref></li>
-			<li><rref>menu</rref></li>
-			<li><rref>menubar</rref></li>
-			<li><rref>radiogroup</rref></li>
-			<li><rref>tree</rref></li>
-			<li><rref>treegrid</rref></li>
-			<li><rref>tablist</rref></li>
-		</ul>
-		<p>More information on managing focus can be found in the <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard">Developing a Keyboard Interface</a> section of the <cite><a href="https://www.w3.org/TR/wai-aria-practices-1.1/"><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Authoring Practices</a></cite> [[WAI-ARIA-PRACTICES-1.1]].</p>
+	  <h2>Managing Focus and Supporting Keyboard Navigation</h2>
+	  <p>An <rref>application</rref> should always have an <a>element</a> with focus when in use as applications require users to have a place to provide user input. All interactive <a>objects</a> should be focusable, and all  parts of composite interactive controls need to be focusable or have a documented alternative method to achieve their function, such as a keyboard shortcut.</p>
+	  <p>When using standard <abbr title="Hypertext Markup Language">HTML</abbr> and basic <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a>widgets</a>, application developers can simply manipulate the tab order or use a script to create keyboard shortcuts to elements in the document. SVG Tiny provides a similar navigation "ring" mechanism that by default follows document order and which should be implemented using system dependent input facilities (the TAB key on most desktop computers). SVG authors can place elements in the navigation order by manipulating the <a href="https://www.w3.org/TR/SVGTiny12/interact.html#FocusableAttribute">focusable</a> attribute and they can dynamically <a href="https://www.w3.org/TR/SVGTiny12/interact.html#specifyingnavigation">specify the navigation order</a> by modifying elements' <a href="https://www.w3.org/TR/SVGTiny12/intro.html#TermNavigationAttribute">navigation attributes</a>.</p>
+	  <p>Usable keyboard navigation in a rich Internet application is different from the tabbing paradigm among interactive elements, such as links and form controls, in a static document. In rich internet applications, the user tabs to significantly complex <a class="termref">widgets</a>, such as a menu or spreadsheet, and uses the arrow keys to navigate within the widget. The changes that <abbr title="accessible rich internet applications">WAI-ARIA</abbr> introduces to keyboard navigation make this enhanced accessibility possible. In <abbr title="accessible rich internet applications">WAI-ARIA</abbr>, any element can be keyboard focusable. In addition to host language mechanisms such as <code>tabindex</code>, <pref>aria-activedescendant</pref> provides another mechanism for keyboard operation. Most other aspects of <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> widget development depend on keyboard navigation functioning properly.</p>
```

Comment:
"Internet" should be lowercase in "Usable keyboard navigation in a rich Internet application..."

Reply:
Fixed.